integration/src/test/java/org/duracloud/integration/durastore/storage/probe/ProbedRestS3Client.java

```diff
@@ -58,16 +57,16 @@
  *
  * @author Bill Branan
  */
-public class ProbedRestS3Client extends AmazonS3Client implements MetricsProbed {
+public class ProbedRestS3Client extends AbstractAmazonS3 implements MetricsProbed {
```

Comment:
This class needs to extend AmazonS3 (the implementation class). This is used by our integration tests with the expectation that calls will flow through to S3. As it is, this causes those integration tests to fail with "UnsupportedOperationException"s.